### DISCUSSION TITLE
Credit for partial use in project

### DISCUSSION THREAD

**Discussion by lostminds** [SlugFiller/godot-vector2d#4] [General]
Hello @SlugFiller,
I'm working on a [2d vector design project](https://paragraphic.design/) using Godot and I'd like to use some (heavily modified) code from this project. More specifically parts of the SVG element parsing code, like all the nice RegEXes you've made. As I understand it from your _Unlicense_ this is fine with you and you're not asking anything in return, but I'd like to include you in the credits as thanks. 
Do you have any preferences on how you'd like to be mentioned (or not) or any specific URL you'd like the credit row to link to? 
Otherwise I'll list it as 
_SVG import parsing code_
[Godot-Vector2d -  SlugFiller](https://github.com/SlugFiller/godot-vector2d)

**Comment by SlugFiller** [comment]
Do it however you like. There are honestly way better SVG parsers out there. Mine only does the bare minimum to work with common SVGs, and would not pass a conformance test. But if it works for you, you can use and reference it however you please.

**Reply by lostminds** [reply]
Yeah, perhaps, but I doubt there are that many that are written in GDScript and I think this will be good enough for the limited support I'll need at least initially. 
I've found what I think are a couple of inconsistencies already in the parsing, do you want me to open issues for things like this I find?

**Reply by SlugFiller** [reply]
[This project](https://github.com/Giwayume/godot-svg) is written in GDScript, and contains a more complete parser. It uses a different approach to rasterization. It uses Expat license, so credit requirement are clear.

You can open issues with the parser if you want, but I'm unlikely to work on this in the foreseeable future, since Godot have specified that first-party support of vector graphics is not on the roadmap, and alternative ways to implement it require features that are not yet available/exposed (Either direct use of the stencil buffer, or generating textures with shaders).

**Reply by lostminds** [reply]
Ok, thanks for the tip. I'll take a look at that project if I run into issues with this one. However, I mostly need the parsing of basic SVG elements and their properties. Rendering and converting these into other entities for use in Godot is something I already have in place. I'll open issues then, just for future reference of anyone looking at this, but I don't expect you to fix them.
